amxmodx/CTask.cpp

```diff
@@ -269,8 +269,10 @@ bool CTaskMngr::taskExists(int iId, AMX *pAmx)
 
 void CTaskMngr::startFrame()
 {
-	for (auto &task : m_Tasks)
+	for(auto i = m_Tasks.length() - 1; i != 0; i--)
```

Comment:
Your loop `i != 0` also skips the first task I believe.
It won't, it caches the length first, so newly appended tasks won't make it into the iteration. However, newly added tasks that occupy an old index will get processed but never executed, but your code (and the old one) have the same flaw so it's not a problem.